nixodmlconverter/__init__.py

```diff
@@ -0,0 +1,12 @@
+import warnings
+
+from sys import version_info as _python_version
+
+from .info import VERSION
+
+if _python_version.major < 3 or _python_version.major == 3 and _python_version.minor < 6:
+    msg = "The '%s' package is not tested with your Python version. " % __name__
+    msg += "Please consider upgrading to the latest Python distribution."
+    warnings.warn(msg)
```

Comment:
Do we want to be more specific about the least python version?
```suggestion
    msg = "... Please consider upgrading to a Python version >= 3.6."
```

Reply:
:+1: for the text change but I am not sure if the suggestion got the correct line. I will push these changes instead.